ethereum/core/src/main/java/org/hyperledger/besu/ethereum/chain/BadBlockReason.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Hyperledger Besu Contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.ethereum.chain;
+
+public enum BadBlockReason {
+  // Standard spec-related validation failures
+  SPEC_VALIDATION_FAILURE,
+  // When an unexpected exception occurs during block processing
+  EXCEPTIONAL_BLOCK_PROCESSING,
```

Comment:
(optional) While you're here - this value is not used anywhere and can be removed.  If you want to leave it for now, I can clean this up in another PR.

Reply:
done